source/server/http/admin.cc

```diff
@@ -544,20 +544,24 @@ AdminImpl::statsAsJson(const std::map<std::string, uint64_t>& all_stats,
   Value histograms_obj;
   histograms_obj.SetObject();
 
-  // It is not possible for the supported quantiles to differ across histograms, so it is ok to
-  // send them once.
-  Stats::HistogramStatisticsImpl empty_statistics;
-  rapidjson::Value supported_quantile_array(rapidjson::kArrayType);
-  for (double quantile : empty_statistics.supportedQuantiles()) {
-    Value quantile_type;
-    quantile_type.SetDouble(quantile * 100);
-    supported_quantile_array.PushBack(quantile_type, allocator);
-  }
-  histograms_obj.AddMember("supported_quantiles", supported_quantile_array, allocator);
+  bool supported_quantiles_added = false;
```

Comment:
nit: I think the logic below would be a bit more clear if you called this variable `found_used_histogram` or something like that.